test/activesupport5/decoding_test.rb

```diff
@@ -10,83 +12,90 @@
 class TestJSONDecoding < ActiveSupport::TestCase
   include TimeZoneTestHelpers
 
+  # Added for testing if Oj is used.
+  test "oj is used as an encoder" do
+    assert_equal ActiveSupport.json_encoder, Oj::Rails::Encoder
+  end
```

Comment:
This doesn't check for a decoder, but the encoder. I'm not sure if that actually tests the proper thing, since there is no `json_decoder` or `json_parser`.

Reply:
Probably good enough. They are both set at the same time.